ravindu14/SliitSpmProject
# Feature/task id #1163 drop down

how to test 
1. go to http://localhost:4200/blank-page url
2. click on drop down list
3. check whether student names available and user should be able to select one of them
